src/java/picard/analysis/CompareMetrics.java

```diff
@@ -0,0 +1,50 @@
+package picard.analysis;
+
+import htsjdk.samtools.metrics.MetricsFile;
+import htsjdk.samtools.util.IOUtil;
+import picard.PicardException;
+import picard.cmdline.CommandLineProgram;
+import picard.cmdline.CommandLineProgramProperties;
+import picard.cmdline.PositionalArguments;
+import picard.cmdline.programgroups.Metrics;
+
+import java.io.File;
+import java.io.FileReader;
+import java.util.List;
+
+/**
+ * Compare two metrics files.
+ */
+@CommandLineProgramProperties(
+        usage = CompareMetrics.USAGE,
+        usageShort = CompareMetrics.USAGE,
+        programGroup = Metrics.class
+)
+public class CompareMetrics extends CommandLineProgram {
+
+    static final String USAGE = "Compare two metrics files";
+
+    @PositionalArguments(minElements = 2, maxElements = 2)
+    public List<File> metricsFiles;
+
+    @Override
+    protected int doWork() {
+        IOUtil.assertFilesAreReadable(metricsFiles);
+        final MetricsFile<?, ?> metricsA = new MetricsFile();
+        final MetricsFile<?, ?> metricsB = new MetricsFile();
+        try {
+            metricsA.read(new FileReader(metricsFiles.get(0)));
+            metricsB.read(new FileReader(metricsFiles.get(1)));
+            final boolean areEqual = metricsA.areMetricsEqual(metricsB) && metricsA.areHistogramsEqual(metricsB);
+            final String status = areEqual ? "EQUAL" : "NOT EQUAL";
+            System.out.println("Files " + metricsFiles.get(0) + " and " + metricsFiles.get(1) + "are " + status);
```

Comment:
The existing convention seems to be that for large CLPs, we use logging; but for small CLPs, we often just write everything to stdout. If people feel strongly I can change it, but there are lots of ```println```'s in existing CLPs.